Sources/IR/IR+RootFieldBuilder.swift

```diff
@@ -113,14 +113,11 @@ class RootFieldBuilder {
       scopePath: LinkedList(rootTypePath)
     )
 
-    await withTaskGroup(of: Void.self) { backgroundTaskGroup in
-      await buildDirectSelections(
-        into: rootIrSelectionSet.selections.direct.unsafelyUnwrapped,
-        atTypePath: rootIrSelectionSet.typeInfo,
-        from: rootSelectionSet,
-        backgroundTaskGroup: &backgroundTaskGroup
-      )      
-    }
+    await buildDirectSelections(
```

Comment:
Thanks for removing this! I added it while I was trying to fix a data race, but once I made the `FieldCollector` and actor, I didn't need it anymore. I must have forgotten to remove it!